From 3d4a64f9655dcd7d03eb7664c64152880a7723f1 Mon Sep 17 00:00:00 2001
From: Maciej Szulik <maszulik@redhat.com>
Date: Wed, 10 May 2023 18:45:13 +0200
Subject: [PATCH] UPSTREAM: <carry>: move test rules from origin

These were brought back in o/o PRs as follows:
- netpol - https://github.com/openshift/origin/pull/26775/
- schedulerpreemption - https://github.com/openshift/origin/pull/27874/
---
 openshift-hack/e2e/annotate/rules.go | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/openshift-hack/e2e/annotate/rules.go b/openshift-hack/e2e/annotate/rules.go
index ce2e8c14913..224551edc0a 100644
--- a/openshift-hack/e2e/annotate/rules.go
+++ b/openshift-hack/e2e/annotate/rules.go
@@ -111,12 +111,6 @@ var (
 			`Netpol \[LinuxOnly\] NetworkPolicy between server and client using UDP should enforce policy based on Ports`,
 			`Netpol \[LinuxOnly\] NetworkPolicy between server and client using UDP should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector`,
 
-			// The new NetworkPolicy test suite is extremely resource
-			// intensive and causes itself and other concurrently-running
-			// tests to be flaky.
-			// https://bugzilla.redhat.com/show_bug.cgi?id=1980141
-			`\[sig-network\] Netpol `,
-
 			`Topology Hints should distribute endpoints evenly`,
 
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1908645
@@ -142,9 +136,6 @@ var (
 
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1953478
 			`\[sig-storage\] Dynamic Provisioning Invalid AWS KMS key should report an error and create no PV`,
-
-			// https://issues.redhat.com/browse/WRKLDS-665
-			`\[sig-scheduling\] SchedulerPreemption \[Serial\] validates pod disruption condition is added to the preempted pod`,
 		},
 		// tests that need to be temporarily disabled while the rebase is in progress.
 		"[Disabled:RebaseInProgress]": {
-- 
2.41.0

